### PR TITLE
OSX build updates, static inline to allow -O0

### PR DESCRIPTION
**Pull request by eam**
Small build tweaks. Ran into some further issues and needed to build -O0, which isn't possible with pure "inline." static inline should work, per: http://www.greenend.org.uk/rjk/tech/inline.html.

Separate PR to follow re the other runtime issues. With this PR I can build successfully.

### CODE REVIEW

**Comment by yahoocla** [comment]
CLA is valid!

**Comment by eam** [comment]
cc @postster

File: Makefile.base [hunk 1]
```diff
@@ -106,7 +110,7 @@ else
   OBJDIR=$(OBJDIR_BASE)
 endif
 LOCAL_LDADD = -L$(TOPDIR)/src/lib/$(OBJDIR)
-COMP_FLAGS +=  -g -O2 -I/usr/include -I$(INCDIR) -I$(TOPDIR)/include -I. -Wall -fPIC 
+COMP_FLAGS +=  -g -O0 -I/usr/include -I$(INCDIR) -I$(TOPDIR)/include -I. -Wall -fPIC
```
Comment (timrc-git) [inline Makefile.base]: Hi Evan, the rest of the changes look good. We should definitely support no-optimization compiles, but for performance reasons, let's not change the default to -O0.
(When the OSX branch is in good shape, I want to merge it into master.)

Comment (eam) [inline Makefile.base]: Ah whoops! Fixing.

**Comment by eam** [comment]
Ok, I think this is now correct

**Pull request merged**